tests/Integration/Infrastructure/Controller/MyArea/Organization/SigningAuthority/EditSigningAuthorityControllerTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Integration\Infrastructure\Controller\MyArea\Organization\SigningAuthority;
+
+use App\Infrastructure\Persistence\Doctrine\Fixtures\OrganizationFixture;
+use App\Tests\Integration\Infrastructure\Controller\AbstractWebTestCase;
+
+final class EditSigningAuthorityControllerTest extends AbstractWebTestCase
+{
+    public function testEdit(): void
+    {
+        $client = $this->login('mathieu.fernandez@beta.gouv.fr');
+        $crawler = $client->request('GET', '/mon-espace/organizations/' . OrganizationFixture::MAIN_ORG_ID . '/signing_authority/edit');
+
+        $this->assertResponseStatusCodeSame(200);
+        $this->assertSecurityHeaders();
+        $this->assertSame('Autorité signataire', $crawler->filter('h2')->text());
+        $this->assertMetaTitle('Autorité signataire - DiaLog', $crawler);
+
+        $saveButton = $crawler->selectButton('Sauvegarder');
+        $form = $saveButton->form();
+
+        // Get the raw values.
+        $values = $form->getPhpValues();
+        $values['signing_authority_form']['name'] = 'Madame la maire';
+        $values['signing_authority_form']['address'] = '3 rue de la Concertation';
+        $values['signing_authority_form']['signatoryName'] = 'Madame la maire';
```

Comment:
(Détail) Pour mettre deux valeurs "réalistes" en s'inspirant de la maquette, comme dans tests/Unit/Application/Organization/SigningAuthority/Command/SaveSigningAuthorityCommandHandlerTest.php

```suggestion
        $values['signing_authority_form']['name'] = 'Madame la maire de Pau';
        $values['signing_authority_form']['address'] = '3 rue de la Concertation';
        $values['signing_authority_form']['signatoryName'] = 'Madame XXX, Maire de Pau';
```